cirrina/server.py

```diff
@@ -425,7 +424,7 @@ def _ws_handler(self, request):
             websocket.close()
             return websocket
 
-        self.websockets.append(websocket)
+        self.websockets[websocket.__hash__()] = websocket
```

Comment:
If an object is [hashable](https://docs.python.org/3/glossary.html#term-hashable) you can use it directly has the key for the dict. No need to call `__hash__`explicitly.